GCTC-NTGC/gc-digital-talent
# Search by qualified classification by the pool instead of expected classification

# :wave: Introduction
This PR will provide the pool id to count the applicants who are all qualified on the selected classification. 
This will also restrict the classification selection  list (NonExecutiveITclassifications) to only that are already existing in the database. 

This PR does NOT change count applicants query or any other. Hence as long as the page passes the api request with correct parameters the accuracy of applicant count doesn't change.  
In this case , the network call should go with correct pool id for the IT classification you selected.  

# ⚡ Impacted Areas
- Classification dropdown
- Estimated Total Count on the right hand side
- Validation for form Submission 

# :test_tube: Testing
-  Create the Pools in local db for ( AS-01 , AS-02, IT-01, IT-02) 
-  Verify the classification filter lists only IT-01 and IT-02 only
-  When you select any classification filter make sure the countApplicants query on the network tab is passed with the correct pool id. 
- Verify on the initial load classification filter loads without any selection 
- Verify the form can be submitted and the request form page is opened
